constant/index.js

```diff
@@ -20,6 +20,9 @@ export const CONFIRMATION_NEEDED = 6;
 export const EMAIL_REGEX = '^[a-zA-Z0-9.!#$%&\'*/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$';
 export const W3C_EMAIL_REGEX = IS_TESTNET ? '.*' : EMAIL_REGEX;
 
+/* TEMP: reformat medium referrer into medium post */
+export const MEDIUM_REGEX = '^(?:https?:\\/\\/)?medium\\.com\\/media\\/[a-zA-Z0-9_]+\\?postId=([a-zA-Z0-9_]+)';
```

Comment:
https://github.com/likecoin/likecoin-store/pull/728/files#diff-0391df65b1d58f7cc015bcd5ef24fb67R89
this one is more generic, since medium support subdomain